theolive/playback/android/00-getting-started.md

```diff
@@ -37,7 +37,7 @@ From here you THEOLive source should be ready to playback in THEOPlayer.
 
 ### Adding Player Controls
 
-At this stage in implementation, you should have a THEOLive stream playing within your Android app. The Player SDK provides functionality for controlling stream playback which can be extended to users in the form of controls or buttons.
+At this stage in implementation, you should have a THEOLive stream playing within your Android app. The Player SDK provides functionality for controlling stream playback which can be extended to users in the form of controls or buttons. The platform also supports an [Open Video UI for the Android SDK](pathname:///open-video-ui/android/getting-started), which provides a component library for advanced styling and control of UI components.
```

Comment:
Don't use `pathname` here, we want to detect (potentially) broken links between articles.
```suggestion
At this stage in implementation, you should have a THEOLive stream playing within your Android app. The Player SDK provides functionality for controlling stream playback which can be extended to users in the form of controls or buttons. The platform also supports an [Open Video UI for the Android SDK](/open-video-ui/android/getting-started), which provides a component library for advanced styling and control of UI components.
```